app/routes.js

```diff
@@ -1,8 +1,10 @@
 const { healthCheck } = require('./controllers/healthCheck'),
-  albumController = require('./controllers/album');
+  albumController = require('./controllers/album'),
+  userController = require('./controllers/user');
 
 exports.init = app => {
   app.get('/health', healthCheck);
   app.get('/albums', [], albumController.getAlbums);
   app.get('/albums/:id/photos', [], albumController.getAlbumPhotos);
+  app.post('/users', [], userController.signUp);
```

Comment:
Remove empty array of middlewares, `app.post('/users', userController.signUp);`, is not necessary